gas-tank/authentication-services.md

```diff
@@ -0,0 +1,111 @@
+## Authentication
+
+### Multiple Address Login
+
+This endpoint facilitates the generation of JWT tokens for multiple wallet addresses in a single request. The request payload should consist of an array of objects, each containing an address and its corresponding signature. Upon successful validation of the signatures, the server will generate JWT tokens for the provided addresses.
+
+::: code-group
```

Comment:
Can you specify what are the messages to sign here (how to get it? via API) and how to sign them?